### PR TITLE
AR Sample code

### PR DESCRIPTION
**Pull request by vanmaxim**

### CODE REVIEW

File: Samples/VisionARSample/VisionARSample/Base.lproj/LaunchScreen.storyboard [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (deniskoronchik) [inline Samples/VisionARSample/VisionARSample/Base.lproj/LaunchScreen.storyboard]: Maybe change path to `Samples/VisionAR`?

File: MapboxVisionAR/VisionARDelegate.swift [hunk 2]
```diff
@@ -9,11 +9,11 @@
 import Foundation
 
 public protocol VisionARDelegate: class {
-    func onARCameraUpdated(visionARManager: VisionARManager, camera: ARCamera)
-    func onARLaneUpdated(visionARManager: VisionARManager, lane: ARLane?)
+    func visionARManager(visionARManager: VisionARManager, didUpdateARCamera camera: ARCamera)
```
Comment (yunikkk) [inline MapboxVisionAR/VisionARDelegate.swift]: Was this really supposed?..

Comment (vanmaxim) [reply MapboxVisionAR/VisionARDelegate.swift]: This is kind a standard naming for delegate in iOS ecosystem

File: Samples/VisionARSample/VisionARSample/Base.lproj/LaunchScreen.storyboard (hunk 1 above)
Comment (vanmaxim) [reply Samples/VisionARSample/VisionARSample/Base.lproj/LaunchScreen.storyboard]: Sure

File: MapboxVisionAR/VisionARDelegate.swift (hunk 2 above)
Comment (yunikkk) [reply MapboxVisionAR/VisionARDelegate.swift]: I mean function renaming `onARCameraUpdated` -> `visionARManager`

Comment (vanmaxim) [reply MapboxVisionAR/VisionARDelegate.swift]: Discussed

**Pull request merged**